### DISCUSSION TITLE
Regarding support of Windows 7 and 8.1 on version 4.6.7 with classic Qt5 build. Can trackers detect? Can you work with the upstream?

### DISCUSSION THREAD

**Discussion by NewEraCracker** [c0re100/qBittorrent-Enhanced-Edition#633] [Q&A] [answered]
With regards to https://github.com/qbittorrent/qBittorrent/issues/21927 and given the fact that qBitTorrent Enhanced Edition 4.6.7 is not an authorized version of qBitTorrent at many trackers. I have some questions to ask:

- Do you change the user-agent of your client, or preserve the same user-agent as the original client?
- Can you work with the upstream to bring a regular build of qBitTorrent version 4.6.7 that supports Windows 7 and 8.1 using Qt5?

Thank you.

**Comment by c0re100** [comment, marked as answer]
No, do it yourself.

**Reply by NewEraCracker** [reply]
PS:

You still have the URLs here:
https://github.com/c0re100/qBittorrent-Enhanced-Edition/blob/v5_0_x/.github/ISSUE_TEMPLATE/config.yml
pointing to the original repository.
